lib/fileHandler/index.js

```diff
@@ -146,6 +148,13 @@ module.exports = function (options) {
 	}
 
 	function s3Upload ({ params, fileGuidName, mimeType, uploadParams, uploadOptions = {} }) {
+
+		// set S3 object ACL so bucket owner can access even if uploader is different AWS account
+		const shouldSetObjectAcl = _.isUndefined(params.bucketOwnerFullControl) ? true : params.bucketOwnerFullControl;
```

Comment:
a) Given the default behaviour is to use ACL, the I think the flag should be negative, so something like `disableBucketAcl`
b) Instead of `isUndefined`, use `isBoolean` to be more explicit about the check

On a related note, can you think of situations where it would be worth overriding/providing a different ACL?

Reply:
Cool have made those changes. Let me know what you think. I don't think we'll need any of the other ACLs, as they would either make the object private, make it public or give the bucket owner read-only access. But I think full control for the bucket owner is the only one we'd want to use in connectors, as that way we have the future option of making the bucket owner the default owner of the object as well (that's a setting on the S3 bucket which is only possible with bucket-owner-full-control).